.github/workflows/ci.yml

```diff
@@ -11,39 +11,74 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    env:
+      NOT_FORK: ${{secrets.NOT_FORK == 'true'}}
+      FORK: ${{secrets.NOT_FORK != 'true'}}
     steps:
     - uses: actions/checkout@v1
+
+    - name: Package Name
+      run: sed -i -e "s/package=\"[^\"]*\"/package=\"com.mathewsachin.fategrandautomata\"/g" AndroidManifest.xml
```

Comment:
How about using a bash if else instead?

I'm not really familiar with bash, but it should be like
```
if [$NOT_FORK == "true"]; then
    PACKAGE_NAME="com.mathewsachin.fategrandautomata"
else
    PACKAGE_NAME="com.mathewsachin.fategrandautomata.fork"
fi
sed -i -e "s/package=\"[^\"]*\"/package=\"$PACKAGE_NAME\"/g" AndroidManifest.xml
```

This will save some copy pasting.

Reply:
PACKAGE_NAME could be shared with the Upload APK step: https://www.edwardthomson.com/blog/github_actions_15_sharing_data_between_steps.html